TransformationSystem/Client/TransformationClient.py

```diff
@@ -2,24 +2,16 @@
 
```

Comment:
The integration test for the TransformatClient -> service -> DB is in https://github.com/DIRACGrid/TestDIRAC/blob/master/Integration/TransformationSystem/TestClientTransformation.py

At a first look it seems OK, but please run the test before merging the PR.